docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -259,12 +261,29 @@ Custom Metrics
 ==============
 You can also pass your own custom metric to relevant checks and suites.
 
-For computer vision the custom metrics should support the
-`Ignite Metric <https://pytorch.org/ignite/metrics.html#how-to-create-a-custom-metric>`__ API.
-
-For tabular metrics the custom metrics should support the
-`sklearn scorer <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.make_scorer.html>`__ API.
-Multiclass classification scorers should assume that the labels are given in a
+For tabular metrics and vision classification tasks the custom metrics function should follow the
+`sklearn scorer <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.make_scorer.html>`__ API, which is
+a function that accepts the parameters: (model, x, y_true), and returns a score with the convention that higher is
+better.
+
+For other computer vision tasks, the custom metrics should be a custom metric class. Deepchecks Custom Metric classes
+are inspired by `Ignite metrics <https://pytorch.org/ignite/metrics.html#how-to-create-a-custom-metric>`__, must
+inherit from :class:`deepchecks.vision.metric_utils.custom_metric.CustomMetric` and implement the following methods:
+``reset``, ``update`` and ``compute``.
+
+    * ``reset`` - resets the metric to its initial state, creates or resets any internal variables.
+    * ``update`` - updates the metric's internal state based on the provided labels and predictions. The method's
+        signature should be ``update(self, output)``, where output is a tuple containing first ``y_pred`` which is the
+        model's output and second ``y_true`` is the ground truth, both given  as lists of numpy objets. For example, for
+        object detection the label would be a list where each element is a numpy array of bounding boxes annotations,
+        and the prediction would be a list where each element is a numpy array of bounding boxes predictions, both in
+        the :doc:`deepchecks format </user-guide/vision/supported_tasks_and_formats.rst>`.
+    * ``compute`` - returns the metric's value based on the internal state. Can be either a single number, or a numpy
+        array of metrics values per class.
```

Comment:
what do you mean by array of metrics values?

Reply:
Tried an alternative phrasing, take a look